### PR TITLE
Add prev also for guest-logging-go

### PR DESCRIPTION
**Pull request by hopkiw**
Same as #39

### CODE REVIEW

File: prow/prowjobs/GoogleCloudPlatform/gcp-guest/gcp-guest-config.yaml [hunk 1]
```diff
@@ -119,3 +119,52 @@ presubmits:
       - name: codecov
         secret:
           secretName: codecov
+
+  GoogleCloudPlatform/guest-logging-go:
```
Comment (krzyzacy) [inline prow/prowjobs/GoogleCloudPlatform/gcp-guest/gcp-guest-config.yaml]: this should live under GoogleCloudPlatform/guest-logging-go subdir

Comment (hopkiw) [reply prow/prowjobs/GoogleCloudPlatform/gcp-guest/gcp-guest-config.yaml]: i'm fine with splitting out but confused why this is the change that prompts it, it's the third repo added to this file. if the dirs under GoogleCloudPlatform should match repo names, then this dir should be renamed from gcp-guest to osconfig, and guest-agent jobs moved out.

if that's what you have in mind, can we do GoogleCloudPlatform/gcp-guest/<repo> dirs, so we can put OWNERS at the gcp-guest level?

Comment (krzyzacy) [reply prow/prowjobs/GoogleCloudPlatform/gcp-guest/gcp-guest-config.yaml]: Technically there's no hard rules, we collect all *.yaml files under the prowjobs/ subdir. I just feel like split out jobs by repos will help you keep your config files reviewable and making more granular changes. Feel free to address it later.

**Comment by krzyzacy** [comment]
/lgtm
/hold

can you rebase and get rid of the first commit (which is already merged)?

**Comment by krzyzacy** [comment]
actually I think upon merging it will drop the commit that's already merged
/this-is-fine
/hold cancel
/approve

**Pull request merged**